TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
He had `-servicing-*` which was causing NuGet to fetch the 1.0.0 version from nuget.org